text/0000-bionic-eos.md

```diff
@@ -0,0 +1,56 @@
+
+## Detailed Explanation
+
+This end of support will entail the following:
+
+### Stacks
+1. Disable the CI that builds the Bionic stacks.
+1. Archive the GitHub repositories for the Bionic stacks (with a note in the README indicating end of support, and with all other content preserved for posterity/discoverability).
+
+### Builders
+1. Disable the CI that builds the Bionic builders.
+1. Archive the GitHub repositories for the Bionic builders (with a note in the README indicating end of support, and with all other content preserved for posterity/discoverability).
+
+### Buildpacks
+1. Stop compiling dependencies for Bionic in buildpacks that currently support them.
+1. Remove Bionic-specific dependencies from the `buildpack.toml` metadata.
+
+### Documentation
+1. Ensure all documentation (and samples) replaces all references to Bionic with Jammy (22.04).
+1. Consider writing a blog post to announce this end of support.
+
+We will leave it up to the relevant maintainers (stacks, builders, language-families, content) to decide on a suitable timeline for completion of the above items. For example, language family maintainers may wish to continue building and including Bionic dependencies even after the Bionic stacks/builders are removed if they believe it provides value to their consumers. This RFC will not be fully implemented until all the above points are addressed, though.
+
+## Rationale and Alternatives
+
+1. Continue to run the stacks/builders CI. This is pointless because there will not be new updates so the CI will be wasting resources checking for updates.
+1. Do not mark the stacks/builders GitHub repositories as end-of-support. This is a poor user experience given the stacks/builders are not being updated. It is misleading to potential consumers.
+1. Continue compiling and distributing buildpack dependencies for Bionic. This is a waste of resources if the buildpack is not updated in the builders and users do not use the stack because it is potentially vulnerable.
+1. Continue to include the dependencies in `buildpack.toml`. This is confusing to buildpack authors and potentially misleading if they aren't being updated/compiled upstream.
+1. Leave documentation/samples/blog as-is. This is a poor user experience for consumers as they will see references to Bionic stacks even though they are not supported
+
+## Implementation
+
+* Update stack/builder READMEs with end-of-support notices and archive the repositories (this will also disable the GitHub actions that build the stacks/builders)
+* Remove Bionic dependency compilation code and GitHub actions for buildpacks that compile their dependencies
+* Remove Bionic dependencies from `buildpack.toml`
+* Update website/samples/blog to remove references to Bionic stacks, builders, and dependencies
+
```

Comment:
Add notes on migrating. Thoughts off the top of my head, which I shared on Slack.

```
Migration from Bionic to Jammy with buildpacks should be straightforward. Run `pack config default-builder paketobuildpacks/builder-jammy-base` (or `-full` or `-tiny`, depending on your preference). This will change the default builder used when you create images with pack.

It is possible to override the default, so check for the following as well:

- In some cases, you might be passing the `--builder` or `-B` flag to `pack build`, which will override the builder for that particular build. In that case, please make sure to update the commands you’re running or scripts to also use the Jammy builders. 

- For Spring Boot users, check your pom.xml or build.gradle files to see if you are specifying a builder (search the file for `builder`), and update those as well.

- One more place to check, look at your CI pipelines to see if you have any which are setting a builder, possibly when running pack, and update those also.
```

Reply:
Also, we really, really need to try and get `pack builder suggest` either a.) updated or b.) redirect the existing stacks to Jammy. We really don't want that to be pointing to old builders. That throws off the new user experience a lot.

Reply:
Pack release should be soon. 🥳

Reply:
I agree with the migration notes. I added a step to include this in the documentation. I didn't put the specifics in because I think that's too detailed for the RFC.